### PR TITLE
feat: adding UIB communication content (#2722 improvement)

### PR DESCRIPTION
**Pull request by marcobonita**
Content of https://github.com/bonitasoft/bonita-doc/pull/2722 + suggested changes

### CODE REVIEW

File: modules/applications/pages/uid-applications-index.adoc [hunk 1]
```diff
@@ -0,0 +1,105 @@
+= Living Applications with Bonita UID
+:description: This page explains what a Living Application is, what it is made of, and how you can create your own applications.
+
```
Comment (benjaminParisel) [inline modules/applications/pages/uid-applications-index.adoc]: As you rename this page, the :page-aliases: are mandatory and refer to the oldest page name.

```suggestion
:page-aliases: applications:custom-applications-index.adoc
```

**Comment by benjaminParisel** [comment]
Are you sure about this preview ? 
image

**Comment by marcobonita** [comment]
> Are you sure about this preview ? image

Weird, this part is identical to the original file

**Review by benjaminParisel — APPROVED** [review]
Fine for me with syntax

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 = Release notes
-:description: Bonita release note
+:description: Bonita release notes for the 2024.3 version
```
Comment (benjaminParisel) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
:description: Bonita release notes for the {bonitaVersion} version
```

**Review by benjaminParisel — APPROVED** [review]

**Comment by benjaminParisel** [comment]
:warning: Waiting for surge issue resolving, to check the preview and then we can merge

**Comment by benjaminParisel** [comment]
@marcobonita Il y a une erreur dans le buld qui dit ceci: 
`[06:24:05.040] FATAL (antora): Page alias cannot reference an existing page: 2024.3@bonita:applications:custom-applications-index.adoc (specified as: applications:custom-applications-index.adoc)
    source: modules/applications/pages/uid-applications-index.adoc in https://github.com/marcobonita/bonita-doc.git (branch: patch-20)`

**Comment by marcobonita** [comment]
> @marcobonita Il y a une erreur dans le buld qui dit ceci: `[06:24:05.040] FATAL (antora): Page alias cannot reference an existing page: 2024.3@bonita:applications:custom-applications-index.adoc (specified as: applications:custom-applications-index.adoc) source: modules/applications/pages/uid-applications-index.adoc in https://github.com/marcobonita/bonita-doc.git (branch: patch-20)`

True, we don't need that alias anymore. I removed it

File: modules/applications/pages/faq.adoc [hunk 3]
```diff
@@ -1,5 +1,146 @@
...
+Unfortunately no. You’ll need to either use Bonita UI Designer or design your interfaces with the language or tool of your choice and integrate them to Bonita. Upgrade to an Access or Scale edition if you want to use Bonita UI Builder.
+
+
+
+=== Is Bonita UI Builder compatible with all supported versions of Bonita?
+Bonita UI Builder is compatible with all Bonita versions starting from 2024.3.
+
+
+
+=== What is an advancedApplication?
```
Comment (rbioteau) [inline modules/applications/pages/faq.adoc]:
```suggestion
=== What is an `applicationLink` ?
```

**Review by rbioteau — CHANGES_REQUESTED** [review]

**Review by rbioteau — APPROVED** [review]

**Pull request merged**